sklearn/metrics/classification.py

```diff
@@ -808,8 +808,8 @@ def precision_recall_fscore_support(y_true, y_pred, beta=1.0, labels=None,
            Godbole, Sunita Sarawagi
            <http://www.godbole.net/shantanu/pubs/multilabelsvm-pakdd04.pdf>`
 
-    Examples
-    --------
+    First example
```

Comment:
I'm confused, "First example" is also followed by a line with dashes, is this not the same form of heading? Or is the heading text that is this issue?